lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -25,7 +25,7 @@ else
   dt_dir="$HOME/src/devtools/devtools-frontend"
 fi
 
-if [[ ! -d "$dt_dir" || ! -a "$dt_dir/front_end/shell.js" ]]; then
+if [[ ! -d "$dt_dir" || ! -a "$dt_dir/front_end/OWNERS" ]]; then
```

Comment:
Was the purpose of this to just check that *some* file existed?

Reply:
some expected file, yes.